pytket/pytket/circuit/display/__init__.py

```diff
@@ -20,14 +20,16 @@
 import time
 import uuid
 import webbrowser
-from typing import Dict, Optional, Union, cast
+from dataclasses import dataclass, field
+from typing import List, Dict, Optional, Union, Literal, cast, Any
```

Comment:
Very inconsequential but in python 3.10+ you can use `list` as a type annotation rather than `List`. Same for `dict` etc. The latest version of pytket requires python 3.10 or newer.

Reply:
updated typings